test/Test/Job.php

```diff
@@ -9,7 +9,7 @@
 
 class Job extends AbstractJob
 {
-    public $performed = false;
+    public bool $performed = false;
```

Comment:
I think typed properties were only introduced with php 7.4, so probably 7.3 would be kind of sad again?